lib/styles/styles.dart

```diff
@@ -59,33 +60,41 @@ class _Padding {
   final xs = 6.0;
 }
 
+class _Spacers {
+  late final _Padding padding;
+
+  get xl => SizedBox.square(dimension: padding.xl);
+  get l => SizedBox.square(dimension: padding.l);
+  get m => SizedBox.square(dimension: padding.m);
+  get s => SizedBox.square(dimension: padding.s);
+  get xs => SizedBox.square(dimension: padding.xs);
+}
+
 class _SeasonSizes {
   final small = 20.0;
   final large = 25.0;
 }
 
 class _ContainerStyles {
+  // Why are these defined here, but others defined ad hoc in app?
```

Comment:
I like having some standard BoxDecorations defined for the app. But it looked like this wasn't consistently used. I'd recommend auditing all the BoxDecorations to see if it can be consolidated further.